filebrowser/run.sh

```diff
@@ -21,10 +21,21 @@ if [ "${DB_PATH}" != "filebrowser.db" ]; then
   DB_FLAG=" -d ${DB_PATH}"
 fi
 
+# Check if filebrowser db exists
+if [ ! -f ${DB_PATH} ]; then
+  filebrowser $DB_FLAG config init >> ${LOG_PATH} 2>&1
+  filebrowser $DB_FLAG users add admin "" --perm.admin=true --viewMode=mosaic >> ${LOG_PATH} 2>&1
+fi
+
+filebrowser $DB_FLAG config set --baseurl=${SERVER_BASE_PATH} --port=${PORT} --auth.method=noauth --root=$ROOT_DIR >> ${LOG_PATH} 2>&1
```

Comment:
```suggestion
  filebrowser $DB_FLAG config init 2>&1 | tee -a "${LOG_PATH}"
  filebrowser $DB_FLAG users add admin "" --perm.admin=true --viewMode=mosaic 2>&1 | tee -a "${LOG_PATH}"
fi

filebrowser $DB_FLAG config set --baseurl=${SERVER_BASE_PATH} --port=${PORT} --auth.method=noauth --root=$ROOT_DIR 2>&1 | tee -a "${LOG_PATH}"
```
Just a suggestion, this will allow showing the output in the Coder UI as well as the log file (useful in case of error).

Reply:
I thought of doing that myself, but decided to stick with the established convention